### PR TITLE
add global dependents

### PR DESCRIPTION
**Pull request by emtudo**
suggestion: https://github.com/GeekyAnts/vue-native-core/pull/4

### CODE REVIEW

**Comment by chovy** [comment]
needs lodash added too.

File: package.json [hunk 1]
```diff
@@ -119,5 +119,8 @@
       "style-loader": "^0.17.0",
       "vue": "2.3.3",
       "vuex": "^2.3.1"
+    },
+    "scripts": {
+      "preinstall": "npm i -g create-react-native-app"
```
[the remark this answers is not in the document]
Comment (emtudo) [reply package.json]: If your installation requires root privileges, you probably installed the node and npm in the wrong way, the correct thing is to install the node for the user, avoiding permissions problems, not root.

[the remark this answers is not in the document]
Comment (neeraj-singh47) [reply package.json]: We'll add this step in readme instead of adding a preinstall script.

[the remark this answers is not in the document]
Comment (emtudo) [reply package.json]: @neeraj-singh47 I reopened the issue then of the readme I had already done.

[the remark this answers is not in the document]
Comment (neeraj-singh47) [reply package.json]: @emtudo We have updated the README to include both crna and react-native-cli. You can close the PR

**Pull request closed without merging**